src/com/bardsoftware/server/auth/AuthServlet.java

```diff
@@ -153,8 +136,36 @@ private String doOauth(HttpApi http, String authProvider, DefaultOAuthPlugin plu
     return doOauthWithCallback(http, plugin, myUrlService.getUrl("oauth.callback", http) + authProvider);
   }
 
+  private String doDevAuth(HttpApi http, DefaultOAuthPlugin plugin) throws IOException {
```

Comment:
Can you move it to line 216 where it used to be? Hopefully the diff will look better